pytorch_lightning/profiler/profilers.py

```diff
@@ -121,14 +122,17 @@ class SimpleProfiler(BaseProfiler):
     def __init__(self, output_filename: Optional[str] = None):
         """
         Args:
-            output_filename (str): optionally save profile results to file instead of printing
+            output_filename (Optional[str]): optionally save profile results to file instead of printing
```

Comment:
```suggestion
            output_filename: optionally save profile results to file instead of printing
```
we don't actually need to write it in the docstring. The sphinx docs will parse the method signature correctly and annotate the docs this way.